README.md

```diff
@@ -72,52 +72,61 @@ This produces:
 <a name="benchmarks"></a>
 ## Benchmarks
 
-As far as we know, it is one of the fastest loggers in town:
+As far as we know Pino is fastest logger in town.
```

Comment:
I would leave this as before. The difference is so tiny, and the approach between bole and pino is essentially the same now.
20ms over 10000 log lines is hardly perceivable.